components/classes/b2b/account-activity.schema.json

```diff
@@ -0,0 +1,71 @@
+{
+  "meta:license": [
+    "Copyright 2024 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/context/account-activity",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "XDM Business Account Activity",
+  "type": "object",
+  "meta:tags": {
+    "b2bSchema": true
+  },
+  "auditable": true,
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:extends": ["https://ns.adobe.com/xdm/data/time-series"],
+  "description": "XDM Business Account Activity is a standard Experience Data Model (XDM) class that tracks the activities related to a business account. This XDM class can only be included in the profile for customers with the B2B or B2P Edition.",
+  "definitions": {
+    "account-activity": {
+      "properties": {
+        "xdm:accountEventKey": {
```

Comment:
Let's remove  this field.
"xdm:accountEventKey": {
So, we will only have:
"xdm:accountKey"
"xdm:accountEventType"